### ISSUE TITLE
(Deno) Cannot set "location"

### ISSUE THREAD

**Issue by luaVolk** [nanojsx/nano#6]
I'm using Deno (1.7.0) and cant use `renderSSR` because `window.location` cant be set
```tsx
import { Component, h, renderSSR } from "https://deno.land/x/nano_jsx@v0.0.14/mod.ts";

class App extends Component {
  render() {
    return (
      <div>ayy</div>
    )
  }
}

renderSSR(<App />);
```
```
error: Uncaught NotSupportedError: Cannot set "location".
        throw new DOMException(`Cannot set "location".`, "NotSupportedError");
              ^
    at set (deno:op_crates/web/12_location.js:340:15)
    at initSSR (<https://deno.land/x/nano_jsx@v0.0.14/ssr.ts>:25:1)
    at renderSSR (<https://deno.land/x/nano_jsx@v0.0.14/ssr.ts>:39:1)
```

**Comment by yandeu** [comment]
Has already been fixed in https://github.com/nanojsx/nano/commit/1e19866af6c958a8797192d8443c3e3d37030649.

Please use deno < 1.7.0 until I release nano jsx v0.0.15.

**Comment by yandeu** [comment]
I just released version v0.0.15 (https://deno.land/x/nano_jsx)

**Comment by CanRau** [comment]
Just wanted to share for future reference that from Deno 1.7 (as I understand) you can "fake" a `location` via the `--location` command line flag
More at https://deno.land/manual@v1.7.0/runtime/location_api
